src/Symfony/Component/Form/Extension/Core/Type/TimezoneType.php

```diff
@@ -72,11 +75,11 @@ public function getBlockPrefix()
     /**
      * Returns a normalized array of timezone choices.
      */
-    private static function getTimezones(int $regions): array
+    private static function getTimezones(int $regions, ?string $country = null): array
     {
         $timezones = array();
```

Comment:
wondering.. should we throw if country is set and regions is not PER_COUNTRY? Or perhaps even better, let OptionsResolver forbid such a pair.

Reply:
Idk, do we really want to enforce this ? I guess if it's not passed then it will not work as expected. OptionsResolver seems a good idea to me.

Reply:
@ogizanagi WDYT ?

Reply:
If it can be easily achieved using the OptionsResolver (and it probably is through normalizers), that would be a better DX indeed :)